templateflow/conf/__init__.py

```diff
@@ -0,0 +1,27 @@
+"""
+Settings
+"""
+from os import getenv
+from pathlib import Path
+from .bids import Layout
+
+TF_DEFAULT_HOME = Path.home() / '.cache' / 'templateflow'
+TF_HOME = Path(getenv('TEMPLATEFLOW_HOME', str(TF_DEFAULT_HOME)))
+TF_GITHUB_SOURCE = 'https://github.com/templateflow/templateflow.git'
+
+if not TF_HOME.exists():
+    try:
+        from datalad.api import install
+        TF_HOME.parent.mkdir(exist_ok=True, parents=True)
+        install(path=str(TF_HOME), source=TF_GITHUB_SOURCE, recursive=True)
```

Comment:
Since you're only catching the `ImportError`, these two lines should go in an `else:` block after the `except` block.